src/content/administration/dashboard.mdx

```diff
@@ -67,3 +67,29 @@ For each namespace, an administrator will be able to:
 The Secrets section of the Admin dashboard lets you define cluster-wide secrets. Using the `$SECRET_NAME` syntax, you can refer to these secrets directly in your Okteto manifest. They are also injected into the [environment running](/docs/cloud/okteto-cli/#built-in-tools-when-deploying-to-okteto-cloud) the commands in the `deploy` section of the manifest.
 
 > Secrets [defined by the user](/docs/cloud/secrets/) take precedence over the secrets defined here in case of a name collision.
+
+We recomment to use [build secrets](/docs/reference/manifest/#build-object-optional) to inject sensitive data into your Dockerfile.
```

Comment:
```suggestion
We recommend using [build secrets](/docs/reference/manifest/#build-object-optional) to inject sensitive data into your Dockerfile.
```

Reply:
I think this belongs in https://www.okteto.com/docs/cloud/secrets/ instead. What do you think @teresaromero ?